test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -1515,6 +1515,183 @@ context('action sidebar', function() {
       .should('contain', 'more comment');
   });
 
+  specify('action comments - show/hide icon in action list items', function() {
+    const testPatient = getPatient();
+
+    const testAction = getAction({
+      relationships: {
+        patient: getRelationship(testPatient),
+        state: getRelationship(stateTodo),
+        comments: getRelationship([], 'comments'),
```

Comment:
I don't think `, comments'` is needed

Reply:
Correct, removed instances of these in https://github.com/RoundingWell/care-ops-frontend/pull/1385/commits/d9e8c49a663caa3d5f4c03a80da4d6105d6e6d6a.